docs/pages/deployment/resource-requirements.rst

```diff
@@ -0,0 +1,20 @@
+.. _resource-requirements:
+
+Resource Requirements
+#####################
+
+The Nuts node is built to be lightweight in terms of CPU and memory usage. Proof: it even runs on a Raspberry PI (Zero).
+
+For a production environment you should be able to run it on a small cloud VM, which typically start at;
+
+- 1 CPU
+- 512 MB RAM
+- 25 GB storage
+
+Required storage depends on network state which grows over time, so make sure to monitor it.
+
+If you use Redis for network state storage the storage requirements will be lower,
+but since search indexes are kept on disk it will still grow over time (although at a lower rate).
+
+Since search indexes use a memory-mapped file memory usage will grow over time as well, although 512 MB is a good starting point.
+When you experience degrading performance when searching, monitor RAM usage and consider upgrading it.
```

Comment:
This {assumes detailed knowledge / provides a lot of details} about the internal workings of the node. Not sure if this is the right place for that. (Random bits of info in the wrong place is likely to result in inconsistent/conflicting documentation in the future.)

Reply:
I'll remove the section, we didn't test the impact of network state to memory usage (and thus search performance) ayways